<launch>

  <!-- DVS-PANO node -->
  <node name="dvs_pano" pkg="dvs_pano" type="dvs_pano" output="screen" args="--v 0">

    <!-- Topics -->
    <param name="events_topic" value="/dvs/events" />
    <param name="camera_info_topic" value="/dvs/camera_info" />
    <param name="events_per_packet" value="1500" />
  </node>  

</launch>
